lib/solr_wrapper/instance.rb

```diff
@@ -204,55 +240,51 @@ def validate!(file)
 
     ##
     # Run a bin/solr command
+    # @param [String] cmd command to run
+    # @param [Hash] options key-value pairs to transform into command line arguments
+    # @param [Array] boolean_flags to include in command (ie. "-c")
+    # @return [StringIO] an IO object for the executed shell command
     # @see https://github.com/apache/lucene-solr/blob/trunk/solr/bin/solr
-    def exec(cmd, options = {})
-      output = !!options.delete(:output)
-      args = [solr_binary, cmd] + solr_options.merge(options).map { |k, v| ["-#{k}", "#{v}"] }.flatten
-
+    # @example start solr in cloud mode on port 8983
+    #   exec('start', {p: '8983'}, ["-c"])
+    def exec(cmd, options = {}, boolean_flags = [])
```

Comment:
I'm not thrilled about adding a third argument just for boolean values. Do you think we could just capture option values that are the boolean true and map them appropriately? We might call this with, e.g.:

```ruby
o.exec(port: 8983, c: true)
```